client/components/platform-checkout/save-user/style.scss

```diff
@@ -0,0 +1,75 @@
+@import '../../../stylesheets/abstracts/colors';
+@import '../../../stylesheets/abstracts/variables';
+
+.platform-checkout-save-new-user-container {
+	display: flex;
+	flex-direction: column;
+	gap: $gap;
+	margin-top: $gap-large;
+
+	h3 {
+		font-weight: 400;
+		font-size: 24px;
+		margin-bottom: 0;
+	}
+
+	.components-checkbox-control__input-container {
+		margin-right: $gap-smaller;
+
+		.components-checkbox-control__input[type='checkbox'] {
+			height: 16px;
+			width: 16px;
+		}
+
+		svg {
+			height: 16px;
+			width: 16px;
+			left: 0;
+			top: 0;
+		}
+
+		label {
+			font-size: 14px;
+		}
+	}
+
+	.save-details-form {
+		display: flex;
+		flex-direction: column;
+		font-size: 16px;
+		line-height: 26px;
+		font-weight: 400;
+		background-color: #f8f8f8;
```

Comment:
Interesting how the colour here is different than the other payment box `#fafafa`. The design does say f8.

Are these UI components something we can use from the `@woocommerce-components` ([link](https://woocommerce.github.io/woocommerce-admin/#/components/))? Maybe we can avoid writing our own CSS?

Reply:
As per our discussion, skipping using the `wc-components` for this section as it does not have the required components. To achieve more consistency with the other part of the page using an existing `class` named `place-order` in this section.